util/audio.py

```diff
@@ -235,38 +251,43 @@ def write_opus(opus_file, audio_format, audio_data):
     opus_file.write(pack_number(width, OPUS_WIDTH_SIZE))
     for i in range(0, len(audio_data), chunk_size):
         chunk = audio_data[i:i + chunk_size]
+        if len(chunk) < chunk_size:
```

Comment:
Maybe add a small comment explaining that we must pad here to prevent Opus from doing bad things?